### PR TITLE
Feature/31 flow assignments

### PR DESCRIPTION
**Pull request by hibes**
This PR adds an Assign() combinator.

### CODE REVIEW

File: tests/test_signal_analog_combinators.py [hunk 1]
```diff
@@ -66,6 +66,13 @@ def test_combinator_not(expr):
     assert str(comb.Not(expr)) == "not {0}".format(str(expr))
 
 
+@given(flows(), flows())
+def test_combinator_assign(assignee, expr):
+    """Assign combinator should always be assignee = expr."""
```
[the remark this answers is not in the document]
Comment (dogonthehorizon) [reply tests/test_signal_analog_combinators.py]: This is in the tests though, and it's consistent with what we've written elsewhere here. I might be misunderstanding what you're getting at here though!

[the remark this answers is not in the document]
Comment (hibes) [reply tests/test_signal_analog_combinators.py]: If this is what you have in mind, I can implement it.  I worry that it's inconsistent with the other tests and methodology, and the testing utilities are pretty solid in telling you what went wrong even with the assert statement. 

```
@given(flows(), flows())
def test_combinator_assign(assignee, expr):
    assignError = "Assign combinator should always be assignee = expr."
    if (not (str(comb.Assign(assignee, expr)) == \
        "{0} = {1}".format(str(assignee), str(expr)))):
        raise ValueError(assignError)
```

[the remark this answers is not in the document]
Comment (venky526) [reply tests/test_signal_analog_combinators.py]: No, you are good. I was just blind. I thought this was in the implementation and not tests. Thanks @dogonthehorizon for enlightening me

**Comment by hibes** [comment]
TODO: Replace Combinator in new documentation entries with Function to reflect the change to Assign's parent class.

**Pull request merged**